src/DevTools/setupTestWrapper.tsx

```diff
@@ -77,7 +81,19 @@ type RTLRenderResult = RenderResult<
   typeof import("@testing-library/dom/types/queries"),
   HTMLElement
 >
-type RenderWithRelay = RTLRenderResult & { env: MockEnvironment }
+
+type RenderWithRelay = RTLRenderResult & {
+  env: MockEnvironment
+  mockResolveLastOperation: (
+    mockResolvers: MockResolvers
+  ) => {
+    operation: OperationDescriptor
+    operationName: string
+    operationVariables: Variables
+  }
```

Comment:
After working with this API a bit while fixing tests, I dropped the extra names from here and started using `getOperationProps(operation)` instead. It felt cleaner but I worried it would create confusion later, while putting these in one single definition as carlos suggested makes clear where they come from without relying on intermediate helper logic. I think the `props` naming can get a little overloaded but otherwise it does end up feeling cleaner for me if we don't mind extra imports (also, i was doing this outside the setupTestWrapper context).

Reply:
For something this exceedingly simple, I'd rather leave everything consolidated around one single single thing. If it grows and actually becomes complex, then we consider breaking things apart, but not prematurely. 

```tsx
const { operationName, operationVariables } = mockResolveLastOperation({ ... })

expect(operationName).toBe("foo")
expect(operationVariables).toBe(bar)
```